src/tools/RenderingTests/main.cpp

```diff
@@ -5,6 +5,7 @@
 
 #include <array>
 #include <cassert>
+#include <cstdio>

```

Comment:
?

Reply:
You remember how we had a bug report a long time ago about how `_vsnwprintf_s` was an undefined symbol? Well as it turns out they were right about it, but it simply didn't happen with the back-then-stable version of VS. In the latest version of VS (or rather MSVC) this code doesn't compile anymore unless we explicitly include `cstdio`, which we probably should've always done.

Reply:
(I fixed it because I used the app to test my change.)